src/pages/Version.tsx

```diff
@@ -179,12 +180,12 @@ export const VersionPage: React.VFC = () => {
         buttons={
           <ActionButtons
             canEnqueueToCommitQueue={canEnqueueToCommitQueue}
-            isPatchOnCommitQueue={isPatchOnCommitQueue}
             canReconfigure={!isPatchOnCommitQueue && isPatch}
+            isPatchOnCommitQueue={isPatchOnCommitQueue}
+            isPatch={isPatch}
```

Comment:
```suggestion
            isPatch={isPatch}
            isPatchOnCommitQueue={isPatchOnCommitQueue}
```
If you were going for alphabetical order!